Sources/StreamChat/Models/UnreadCount.swift

```diff
@@ -11,20 +11,25 @@ public struct UnreadCount: Decodable, Equatable {
         case userUnreadCount = "user"
         case channels = "unread_channels"
         case messages = "total_unread_count"
+        case threads = "unread_threads"
     }
 
-    /// The default value representing no unread channels and messages.
-    public static let noUnread = UnreadCount(channels: 0, messages: 0)
+    /// The default value representing no unread channels, messages and threads.
+    public static let noUnread = UnreadCount(channels: 0, messages: 0, threads: nil)
```

Comment:
why not 0?

Reply:
I initially had 0, then switched to `nil` because I was thinking about customers using this to check if there are unreads or not. Hopefully they are not, and this should only be used for initializations. Because if a customer was previously using this to check if there are reads or not, this is breaking. But it is breaking either if we set it to `nil` or `0`. So I switched back to 0 because it makes more sense.

Reply:
What I mean by breaking is that, if somehow a customer is doing something like:
```
if unreadCount == .noUnread {
   // do something
} 
```

This can stop working, because most likely the customer will have now unread threads, and some customers won't care about the threads for now

Reply:
Maybe we could even deprecated this `noUnread` ? 🤔 It is not a good idea at all

Reply:
Yes, but the property is misleading, it is not clear what "noUnread" means: Do you count all the messages? Do you count channels and messages? Do you also count threads? It is just better to access the properties directly IMO

Reply:
For now I'll keep it like this, but on the next PR I'll revisit this